src/test/obj_pool_lookup/obj_pool_lookup.c

```diff
@@ -123,6 +137,18 @@ main(int argc, char *argv[])
 
 		MUNMAP(guard_after[i], Ut_pagesize);
 	}
+
+	for (int i = 0; i < npools; ++i) {
+		UT_ASSERTeq(pmemobj_pool_by_oid(oids[i]), NULL);
+
+		define_path(path, length, dir, i);
+		pops[i] = pmemobj_open(path, LAYOUT_NAME);
```

Comment:
UT_ASSERTne(pops[i], NULL)

Without that test will succeed when pmemobj_open will dail.